### ISSUE TITLE
Bump to edk2-stable202111

### ISSUE THREAD

**Issue by Harvester57** [pbatard/UEFI-Shell#2]
Hi !

A new version of the upstream shell was released on 26 Nov. 2021, could you update your module reference ?

https://github.com/tianocore/edk2/releases/tag/edk2-stable202111

(I can't clone or push to GitHub from work, I will try to open a PR when I get back home if you haven't updated the repo yet :))

Thanks !

**Comment by pbatard** [comment]
I know there's been a new stable release of EDK2, but I'm not planning to publish a shell for every quarterly EDK2 stable.

That's the reason why I settled on H1/H2 (H for "half-year") instead of Q for the tags.

The idea behind not multiplying releases is because I'm integrating this into [FIDO](https://github.com/pbatard/Fido) and if I were to do quarterly, I'd have to contend with a list of entries that is becoming a bit too long for my taste after a few years (and yeah, I know I could forcefully remove entries after a while), and also, more importantly, because I'm deliberately sticking to the naming scheme of Windows releases, where we usually only get 2 releases per year (such as 21H1 and 21H2).

The idea is that users coming from FIDO will be used to seeing YYHH for the Windows releases, and I want to make the UEFI Shell releases not too dissimilar, so as to keep the naming scheme familiar across the board.

As a result, I am __not__ planning to produce more than 2 UEFI Shell releases per year, which means that, since we have already gone through 21H2, the earliest UEFI Shell update I'm going to push next would be for the 2022.03 EDK2 stable... or, most likely, 2022.05, since I've more or less planned to settle on using YYYY.05 and YYYY.11 to match the usual months of OS releases/updates from Microsoft.

Plus it gives me an excuse not to have to tend to this repository too frequently, even as I expect you to protest that bumping to stable should take less than a minute (though I'm going to dispute that things always go that smoothly, especially when you throw a brand new toolchains like RISC-V into the mix, considering that I had to spend a lot of time troubleshooting GitHub Actions for it, and I very much expect more breakage in the future).

In short, with this repository, I am interested in stability rather than the latest and greatest from EDK2. Therefore, half-yearly updates to the repo, even as we have quarterly EDK2 updates, is all you're going to find here (but you are of course free to fork this repo and produce your own YYQQ releases in your fork).

&rarr; Rejected.
